gravitee-am-service/src/main/java/io/gravitee/am/service/reporter/builder/AuditBuilder.java

```diff
@@ -158,7 +167,7 @@ public T oldValue(Object oldValue) {
 
     public T principal(User principal) {
         if (principal != null) {
-            setActor(principal.getId(), EntityType.USER, principal.getUsername(), getDisplayName(principal), getReferenceType(principal), getReferenceId(principal));
+            setActor(principal.getId(), EntityType.USER, principal.getUsername(), getDisplayName(principal), getReferenceType(principal), getReferenceId(principal), null, null);
```

Comment:
Will it not be better to add a new method parameters for setActor and setTarget with these two extra information ? 

I'm not a big fan of having (null, null) as parameters everywhere. 

Thanks

Reply:
fixed